src/scripts/main.js

```diff
@@ -3,6 +3,25 @@
 const people = require('./lib/people');
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+console.log(people);
 
-// write your code here
+people.forEach(item => {
+  const age = item.died - item.born;
+
+  const century = Math.ceil(item.died / 100);
+
+  const tr = document.createElement('tr');
+
+  tr.innerHTML = `
+  <td>${item.name}</td>
+  <td>${item.sex}</td>
+  <td>${item.born}</td>
+  <td>${item.died}</td>
+  <td>${age}</td>
+  <td>${century}</td>
+  `;
+
+  const a = document.querySelector('.dashboard');
```

Comment:
Give more descriptive name 
```suggestion
  const dashboard = document.querySelector('.dashboard');
```